tfhe/src/integer/server_key/radix_parallel/tests_unsigned/test_vector_find.rs

```diff
@@ -405,15 +407,15 @@ where
 
     // We want to split test in half,
     // one half where the lut contains the clear, the other half where it does not
-    const HALVED_NB_TEST: usize = if NB_TESTS > 1 { NB_TESTS / 2 } else { 0 };
+    let halved_nb_test: usize = if nb_tests > 1 { nb_tests / 2 } else { 0 };
 
     let mut lut = Vec::with_capacity(modulus as usize);
-    for i in 0..NB_TESTS {
+    for i in 0..nb_tests {
         lut.clear();
 
         let clear = rng.gen_range(0..modulus);
         let num_values = rng.gen_range(1..MAX_VEC_LEN) as usize;
-        let occurrence_count = if i < HALVED_NB_TEST { 0 } else { 1 };
+        let occurrence_count = if i < halved_nb_test { 0 } else { 1 };
```

Comment:
I propose replacing the by 
```rust
for i in 0..2*nb_tests {

let occurrence_count = if i < nb_tests { 0 } else { 1 };
```

to fix the case nb_tests=1 when tarpaulin is enabled

Reply:
I don't really like this because it means doubling the test for non tarpaulin cases and these tests are already long (1h and more) so doubling them is just a no go

What we can do is `nb_tests.min(2)` to ensure 2 tests

Reply:
And since the code is FHE, there is no branching between the two cases, so even with only one nb_tests all code path will be executed

Reply:
Ok, all good then!